[2024-06-26T07:02:18.962+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-26T07:02:19.047+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_for_alerting_final.check_data_in_stream scheduled__2024-06-26T07:01:00+00:00 [queued]>
[2024-06-26T07:02:19.080+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_for_alerting_final.check_data_in_stream scheduled__2024-06-26T07:01:00+00:00 [queued]>
[2024-06-26T07:02:19.082+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-26T07:02:19.132+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): check_data_in_stream> on 2024-06-26 07:01:00+00:00
[2024-06-26T07:02:19.151+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4670) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-26T07:02:19.156+0000] {standard_task_runner.py:63} INFO - Started process 4671 to run task
[2024-06-26T07:02:19.158+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_for_alerting_final', 'check_data_in_stream', 'scheduled__2024-06-26T07:01:00+00:00', '--job-id', '4168', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_for_alerting.py', '--cfg-path', '/tmp/tmp3da8bd2n']
[2024-06-26T07:02:19.162+0000] {standard_task_runner.py:91} INFO - Job 4168: Subtask check_data_in_stream
[2024-06-26T07:02:19.323+0000] {task_command.py:426} INFO - Running <TaskInstance: demo_dag_for_alerting_final.check_data_in_stream scheduled__2024-06-26T07:01:00+00:00 [running]> on host ced2311e92d2
[2024-06-26T07:02:19.682+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo_dag_for_alerting_final' AIRFLOW_CTX_TASK_ID='check_data_in_stream' AIRFLOW_CTX_EXECUTION_DATE='2024-06-26T07:01:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-26T07:01:00+00:00'
[2024-06-26T07:02:19.685+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-26T07:02:19.760+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-26T07:02:19.762+0000] {helper_functions.py:88} INFO - Reading encrypted details from ('/opt/***/dags/postgres_connection.py', '/opt/***/dags/***_api_configuration.py')
[2024-06-26T07:02:19.787+0000] {helper_functions.py:91} INFO - Decrypted postgres connection: {'user': '***', 'password': '***', 'dbname': '***', 'host': '172.240.4.3', 'port': '5432'}
[2024-06-26T07:02:19.800+0000] {helper_functions.py:58} INFO - Executing create table SQL
[2024-06-26T07:02:19.806+0000] {helper_functions.py:62} INFO - Checking existing record for dag_id and task_id
[2024-06-26T07:02:19.813+0000] {helper_functions.py:79} INFO - Inserting new record for demo_dag_for_alerting_final - check_data_in_stream
[2024-06-26T07:02:19.814+0000] {helper_functions.py:104} ERROR - Error interacting with the database: can't execute an empty query
[2024-06-26T07:02:19.820+0000] {python.py:237} INFO - Done. Returned value was: insert_new_record_into_audit_table
[2024-06-26T07:02:19.822+0000] {branch.py:36} INFO - Branch into insert_new_record_into_audit_table
[2024-06-26T07:02:19.824+0000] {skipmixin.py:178} INFO - Following branch insert_new_record_into_audit_table
[2024-06-26T07:02:19.896+0000] {skipmixin.py:246} INFO - Skipping tasks [('skipped_due_to_data_unavailability', -1)]
[2024-06-26T07:02:20.005+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-26T07:02:20.073+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=demo_dag_for_alerting_final, task_id=check_data_in_stream, run_id=scheduled__2024-06-26T07:01:00+00:00, execution_date=20240626T070100, start_date=20240626T070219, end_date=20240626T070220
[2024-06-26T07:02:20.143+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-26T07:02:20.282+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-26T07:02:20.289+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
